accord-core/src/main/java/accord/coordinate/CoordinationAdapter.java

```diff
@@ -310,14 +302,6 @@ private static class AsyncInclusiveSyncPointAdapter<U extends Unseekable> extend
             protected AsyncInclusiveSyncPointAdapter() {
                 super();
             }
-
-            @Override
-            protected void addOrExecuteCallback(ExecuteBlocking<U> execute, BiConsumer<? super SyncPoint<U>, Throwable> callback)
-            {
-                // If this is the async adapter then we want to invoke the callback immediately
```

Comment:
what is the reason to make this async rather than immediate?

Reply:
I just properly separated the logic for async/blocking, since we shouldn't be executing a blocking coordination act all for async, rather than simply sharing everything except deciding in `addOrExecuteCallback` whether a/sync

Reply:
ok ill need to look closer, I missed that detail